### ISSUE TITLE
Support :browse-namespace command

### ISSUE THREAD

**Issue by zjhmale** [zjhmale/vscode-idris#30]

**Comment by zjhmale** [comment]
The `:browse-namespace` also need some enhancement, since there is not location information (file-path:line:column) with the result.

**Comment by philipcraig** [comment]
Could who-calls be wired up to the Extension Find All References command, or does that not make sense?

**Comment by zjhmale** [comment]
Yeah, we don't really need some commands here, and some concrete commands still need some improvement itself. Thanks for the heads up.
>Could who-calls be wired up to the Extension Find All References command, or does that not make sense?

**Comment by zjhmale** [comment]
BTW, `who-calls` command may be useful to help use implementing the `find-usage` feature.

**Comment by st3v3c00k** [comment]
I cannot see any keyboard shortcuts for the existing commands.  Are they planned?

**Comment by zjhmale** [comment]
Hi @st3v3c00k, there used to be a default key binding for all the existing commands but then I removed them in order to avoid introducing any key binding conflicts. So you can call them using right click dropdown menu, using command palette or add command key bindings by you self in `keybindings.json` setting file.

**Comment by st3v3c00k** [comment]
Thanks.
